symfony/routing/5.1/config/packages/routing.yaml

```diff
@@ -0,0 +1,11 @@
+framework:
+    router:
+        utf8: true
+
+        # Replace these nodes to configure the "request context" to use to generate URLs
```

Comment:
Configure how to generate URLs in non-HTTP contexts, such as CLI commands.

Reply:
Shorter is better, thx.